src/app/register/register.component.html

```diff
@@ -0,0 +1,92 @@
+<div id="banner"></div>
+<main>
+  <div class="register">
+    <biggive-page-section>
+      <div class="heading-wrapper">
+        <biggive-heading
+          space-above=0
+          space-below=6
+          html-element="h1"
+          size="1"
+          align="center"
+          text="Donor Registration"
+        ></biggive-heading>
+      </div>
+      <re-captcha
+        #captcha
+        size="invisible"
+        errorMode="handled"
+        (resolved)="captchaReturn($event)"
+        (errored)="captchaError()"
+        siteKey="{{ recaptchaIdSiteKey }}"
+      ></re-captcha>
+
+      <p>
+        Registering as a donor with Big Give will help you donate to multiple campaigns, and will you the option
+        to donate by bank transfer as an alternative to using a payment card.
+      </p>
+      </biggive-page-section>
+      <biggive-page-section>
+
+        <form
+          [formGroup]="registrationForm"
+          (keydown.enter)="register()"
+        >
+
+          <biggive-text-input spaceBelow="4">
+            <label slot="label" for="firstName">First Name *</label>
+            <input
+              slot="input"
+              matInput type="text" id="firstName" formControlName="firstName">
+          </biggive-text-input>
+          <biggive-text-input spaceBelow="4">
+            <label slot="label" for="lastName">Last Name *</label>
+            <input
+              slot="input"
+              matInput type="text" id="lastName" formControlName="lastName">
+          </biggive-text-input>
+          <biggive-text-input spaceBelow="4">
+            <label slot="label" for="emailAddress">Email address *</label>
+            <input
+              slot="input"
+              matInput type="email" id="emailAddress" formControlName="emailAddress" autocapitalize="off">
+          </biggive-text-input>
+          <biggive-text-input spaceBelow="4">
+            <label slot="label" for="password">Password *</label>
+            <input
+              slot="input"
+              matInput type="password" id="password" formControlName="password">
```

Comment:
We could add a "repeat password" input, but I think it's probably not necessary, and we don't have one on the existing registration modal. See https://ux.stackexchange.com/questions/484/is-a-repeat-password-field-necessary-in-a-signup-page

Although we're not requiring the email address to be confirmed before the new account is used, so if someone's email doesn't work and they mistype their password they will have no way to recover the account.